sdk/rust/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "dstack-rust"
+version = "0.1.0"
+edition = "2024"
+
+[dependencies]
+alloy = { version = "0.15", features = ["signers", "signer-local"] }
+hex = "0.4"
+http = "0.2"
+http-client-unix-domain-socket = "0.1.1"
+reqwest = { version = "0.11", features = ["json"] }
+serde = { version = "1.0", features = ["derive"] }
+serde_json = "1.0"
+sha2 = "0.10.9"
+
+[dev-dependencies]
+evidence_api = { git = "https://github.com/cc-api/evidence-api.git", branch = "main" }
```

Comment:
Suggest to change: use [dcap-qvl](https://github.com/Phala-Network/dcap-qvl) instead. We opened an issue (https://github.com/cc-api/evidence-api/issues/143) and submitted a PR (https://github.com/cc-api/evidence-api/pull/144) to evidence-api, but it seems no one is maintaining it anymore.